### ISSUE TITLE
setOpacity on RasterLayers

### ISSUE THREAD

**Issue by Saulzi** [Esri/esri-leaflet#908]
* Browser and version:

ie: NA

* Version of Leaflet (`L.version`):

1.02

* Version of esri Leaflet (`L.esri.VERSION`):

2.0.6

Steps to reproduce the error:

1. Create a layer which inherits RasterLayer
2. Immediatly set its opacity

What happens is A exception is raised because this._currentImage is null

I was expecting everything to be fine and dandy.

fix is to wrap this._currentImage in if statement.

**Comment by jgravois** [comment]
99% sure this was fixed in #903.

**Comment by Saulzi** [comment]
Nope. basically issue is in rasterlayer.js

```js
  setOpacity: function (opacity) {
    this.options.opacity = opacity;
    this._currentImage.setOpacity(opacity);
    return this;
  },
```
should be
```js
  setOpacity: function (opacity) {
    this.options.opacity = opacity;
    if (this._currentImage)
    {
        this._currentImage.setOpacity(opacity);
     }
    return this;
  },
```

I will shortly have a pr with fix and test,

**Comment by jgravois** [comment]
> I will shortly have a pr with fix and test.

that would be fantastic. :cookie: thanks so much!

**Comment by jgravois** [comment]
resolved by 10666308c018888f614974284a91c5f743d56eba
